### PR TITLE
added fpm to the kiss repos

### PR DESCRIPTION
**Pull request by FultonBrowne**
Please fill out the template.

Read: https://k1ss.org/wiki/kiss/style-guide

---

## Description of package

fpm is a very basic power management utility that's fast, reliable and very small (24 lines of code) https://github.com/FultonBrowne/fpm

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/fpm/version
/var/db/kiss/installed/fpm/sources
/var/db/kiss/installed/fpm/manifest
/var/db/kiss/installed/fpm/checksums
/var/db/kiss/installed/fpm/checksum
/var/db/kiss/installed/fpm/build
/var/db/kiss/installed/fpm/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/bin/fpm
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/fpm/build [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -e
```
Comment (dylanaraps) [inline community/fpm/build:1]: Add a newline here.

File: community/fpm/build [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+install -Dm 755 fpm "$1/usr/bin/fpm"
```
Comment (dylanaraps) [inline community/fpm/build:2]: Drop space between m and 7.

**Comment by FultonBrowne** [comment]
@dylanaraps I fixed the styling issues

**Comment by dylanaraps** [comment]
I had a look at the source code and am a little conflicted. This is a tiny shell script, really tiny. I don't think this has any business being a package in Community. The script also uses `#!/bin/sh` as the shebang while being partially written in bash. I also recommend running shellcheck on the code as I can see a myriad of issues. :+1:

**Pull request closed without merging**